src/tests/Lotto.js

```diff
@@ -0,0 +1,34 @@
+import { ConstantNumbers } from './constant';
+
+const createLottoNumbersArray = () => {
+	return Array.from(
+		{ length: ConstantNumbers.MAX_NUMBER - ConstantNumbers.MIN_NUMBER + 1 },
+		(_, i) => i + ConstantNumbers.MIN_NUMBER
+	);
+};
+class Lotto {
+	static price = ConstantNumbers.LOTTO_PRICE;
```

Comment:
```suggestion
	static PRICE = ConstantNumbers.LOTTO_PRICE;
```
이 부분도 상수이기 때문에 이렇게 대문자로 표현해주면 좋겠네요!